test_conformance/gles/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ set (${MODULE_NAME}_SOURCES
         ../../test_common/gles/helpers.cpp
     )
 
-if(ANDROID)
-    list(APPEND CLConform_LIBRARIES GLESv2)
+if(ANDROID OR UNIX)
+    list(APPEND CLConform_LIBRARIES EGL GLESv2)
 elseif(WIN32)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGLES3")
     list(APPEND CLConform_LIBRARIES libEGL libGLESv2 )
```

Comment:
Could we get rid of the "lib" prefix on the WIN32 case to match Android and Unix? (The build will still use "libLibrary.so" internally) This allows vendors to provide a CMake library object with one name via the CMakeVendors.txt file.

We have a windows development platform that compiles against GLESv2. To support both GLESv2 and GLESv3 on various implementations and to unify across various OSs, can we make the "-DGLES3" flag depend on a CMAKE cache variable?